service/src/test/java/com/sedliarov/learningtable/integration/StudentControllerIntegrationTests.java

```diff
@@ -0,0 +1,202 @@
+
+    // then
+    assertThat(student.getStatusCode()).isEqualTo(HttpStatus.OK);
+    assertThat(student.getBody()).isEqualTo(studentMapper);
+  }
+
+  @Test
+  void negativeTestGetStudentById() {
+
+    // given
+    Student newStudent = StudentFixture.createEntityWithId(UUID.fromString("3e1e6d16-451b-4748-b6a0-8f4a84a0a53a"));
+    Student savedStudent = studentRepository.save(newStudent);
+    StudentDto studentMapper = mapper.entityToDto(newStudent);
+
+    // when
+    ResponseEntity<StudentDto> student =
+        exchangeGetWithoutAuth(STUDENTS_URL + savedStudent.getStudentId(), StudentDto.class);
+
+    // then
+    assertThat(student.getStatusCode()).isEqualTo(HttpStatus.OK);
+    assertThat(student.getBody()).isNotEqualTo(studentMapper);
+  }
+
+  @Test
+  void testGetStudents() {
+
+    // given
+    Student student = studentRepository.save(StudentFixture.createEntityWithFirstAndSecondName("Aria", "Arievna"));
+    Student student1 = studentRepository.save(StudentFixture.createEntityWithFirstAndSecondName("Aria1", "Arievna1"));
+    List<StudentDto> expectedStudents = List.of(mapper.entityToDto(student), mapper.entityToDto(student1));
+
+    // when
+    ResponseEntity<StudentDto[]> response = exchangeGetWithoutAuth(STUDENTS_URL, StudentDto[].class);
+
+    // then
+    assertThat(response.getStatusCode()).isEqualTo(HttpStatus.OK);
+    assertThat(response.getBody()).containsExactlyElementsOf(expectedStudents);
+  }
+
+  @Test
+  void negativeTestGetStudents() {
+
+    // given
+    List<StudentDto> expectedStudents = new ArrayList<>();
+
+    // when
+    ResponseEntity<StudentDto[]> response = exchangeGetWithoutAuth(STUDENTS_URL, StudentDto[].class);
+
+    // then
+    assertThat(response.getStatusCode()).isEqualTo(HttpStatus.OK);
+    assertThat(response.getBody()).containsExactlyElementsOf(expectedStudents);
+  }
+
+  @Test
+  void testCreateStudent() {
+
+    // given
+    StudentDto newStudent = StudentFixture.createDto();
+
+    // when
+    ResponseEntity<StudentDto> student =
+        exchangeAddWithoutAuth(STUDENTS_URL, newStudent, StudentDto.class);
+
+    // then
+    assertThat(student.getStatusCode()).isEqualTo(HttpStatus.CREATED);
+    assertThat(student.getBody().getFirstName()).isEqualTo(newStudent.getFirstName());
+    assertThat(student.getBody().getSecondName()).isEqualTo(newStudent.getSecondName());
+    assertThat(student.getBody().getNote()).isEqualTo(newStudent.getNote());
+    assertThat(student.getBody().getGroup()).isEqualTo(newStudent.getGroup());
+  }
+
+  @Test
+  void negativeTestCreateStudent() {
+
+    // given
+    StudentDto newStudent = StudentFixture.createDto();
+
+    // when
+    ResponseEntity<StudentDto> student =
+        exchangeAddWithoutAuth(STUDENTS_URL + "/" + newStudent.getStudentId(), newStudent, StudentDto.class);
+
+    // then
+    assertThat(student.getStatusCode()).isEqualTo(HttpStatus.METHOD_NOT_ALLOWED);
+  }
```

Comment:
Bad example. Are you sure that you testes needed stuff? try to create existing user